src/organization/configuration/identity-providers/org-settings-identity-providers.component.scss

```diff
@@ -65,6 +65,14 @@
     margin-bottom: 16px;
   }
 
+  &__search-bar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    // Magic value to have proper alignment and consistent visual experience with users screen
```

Comment:
Isn't it the same as here? 
https://github.com/gravitee-io/gravitee-management-webui/blob/b76ee6503c1d2440b68b320503e73b6eacf0c9da/src/organization/configuration/users/org-settings-users.component.scss#L7

Reply:
Not exactly because we do not have a search input, we need to discuss with Greg on these specific things